cli/src/config/merge_tools.toml

```diff
@@ -11,6 +11,15 @@ merge-args = ["$left", "$base", "$right", "-o", "$output", "--auto-merge"]
 program="meld"
 # If using this as a template, note that `$output` is repeated twice below
 edit-args = ["$left", "$output", "$right", "-o", "$output"]
+# With Meld's 3-way merge resolution mode, once the output pane has been
```

Comment:
Perhaps saying "With the normal Meld merge conflict resolution setup,..." would be a bit clearer.